ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1386,18 +1413,18 @@ and Danube Cloud installation.\n\n"
 
 	printheader "Networking"
 	message="
-To set up networking you must first configure a network tag. A network tag
+To set up networking you must first configure network tags. A network tag
 refers to a physical NIC or an aggregation. Virtual machines will be created on
-top of a network tag. Setup will first create a network tag and configure a NIC
-so that you can access the Danube Cloud node. After setup has been completed,
-you will have the option of creating additional network tags and configuring
-additional NICs for accessing the compute node through the nictagadm command.
+top of a network tag. Setup will first create an admin network tag and
+configure a NIC so that you can access the Danube Cloud node. Then you will
+have the option of creating additional network tags and configuring
+additional NICs for accessing the compute node or virtual machines.
```

Comment:
The message is OK, but I think that we should provide the user with the fact that she can configure NIC tags with the nictagadm command. I suggest to add one sentence about the possibility to configure more nictags later with the nictagadm command.